Sources/NIOCore/ByteBuffer-hexdump.swift

```diff
@@ -85,6 +95,46 @@ extension ByteBuffer {
         let endHex = back.hexDumpPlain()
         return startHex + " ... " + endHex
     }
+    
+    /// Return a `String` of  hexadecimal digits of the readable bytes in the buffer,
+    /// analog to `.plain` format but without whitespaces.
+    /// `hexDumpCompact()` always dumps all readable bytes, i.e. from `readerIndex` to `writerIndex`,
+    /// so you should set those indices to desired location to get the offset and length that you need to dump.
+    private func hexDumpCompact() -> String {
+        var hexString = ""
+        hexString.reserveCapacity(self.readableBytes * 3)
```

Comment:
Drive by comment. Is this meant to be `3`? 
```suggestion
        hexString.reserveCapacity(self.readableBytes * 2)
```